src/Symfony/Component/Config/Definition/ArrayNode.php

```diff
@@ -209,8 +209,8 @@ public function getDefaultValue()
      */
     public function addChild(NodeInterface $node)
     {
-        $name = $node->getName();
-        if (!strlen($name)) {
+        $name = (string) $node->getName();
```

Comment:
@xabbuh : I can just revert this change and create a bug-report, what do you think?